app/models/ticket.rb

```diff
@@ -124,6 +126,14 @@ def set_default_notifications!
     self.notified_user_ids = users.map(&:id)
   end
 
+  def is_unread?(user)
+    unread_users.include? user
```

Comment:
While this feature is pretty nice in principle, this change is making Brimir very slow to render for us (20+ seconds) on a system with some 100 open tickets and roughly 2000-3000 users, as this is triggering n+ slow queries per ticket when displaying them.